platform-hub-api/spec/factories/allocations.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :allocation do
+    
```

Comment:
This factory need a body.

Reply:
I suppose I could provide default items for the two different polymorphic associations, but I wanted to leave this up to the consumer of the factory. In which case there's nothing left to put in the body (but still need a factory).